Jenkinsfile

```diff
@@ -88,6 +88,26 @@ pipeline {
                         echo 'Tests done!'
                     }
                 }
+                stage('Calculate Coverage') {
```

Comment:
we should see if we can move this to its own stage? so its

check/rustfmt then tests then coverage

Reply:
We can, but what's the advantage of that? Wouldn't that just increase the build time?

Reply:
only reason I see is if codecov ratelimits us and we prob dont wanna post coverage unless tests pass?

Reply:
but I dont mind either way.

Reply:
I don't think it's gonna be a problem, so let me revert the broken commit I just pushed.